applications/nvidia_nim/nvidia_nim_chat/metadata.json

```
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "Chat with NVIDIA Inference Microservice (NIM)",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.0",
+			"tested_versions": [
+				"1.0.0"
+			]
```

Can you confirm the tested version? Looks like 2.1.0 based on `requirements.txt`:
```suggestion
			"minimum_required_version": "2.1.0",
			"tested_versions": [
				"2.1.0"
			]
```

---

This app should work with 1.0. I will confirm.

---

I have tested the app working using 1.0.3 of the PyPI wheel. Thanks!